app/views/admin/reports/show.html.haml

```diff
@@ -7,6 +7,9 @@
   - else
     = link_to t('admin.reports.mark_as_unresolved'), reopen_admin_report_path(@report), method: :post, class: 'button'
 
+- unless @report.account.local? || @report.target_account.local?
```

Comment:
A convenience method like `@report.remote_content?` (or `forwarded_report?` or something) might be nice readability-wise here, but not necessary.

Reply:
I'm giving it some thoughts but I think both `forwarded_report?` could be misleading as a local report can be forwarded to a remote server (we already have `forwarded?` for that), `remote_content?` could make sense, but it could be a local report about remote content. I think there is just too much ambiguity.